recipe/meta.yaml

```diff
@@ -42,6 +42,11 @@ test:
 about:
   home: http://github.com/inducer/islpy
   license: MIT
+  license_file:
+    - doc/misc.rst
```

Comment:
@inducer, is this the correct license file for islpy?

Reply:
Yup.